packages/components/text/src/text.vue

```diff
@@ -15,6 +29,8 @@ defineOptions({
 })
 
 const props = defineProps(textProps)
+const slots = useSlots()
+const emits = defineEmits(['ellipsis'])
```

Comment:
## Suggest

It is recommended to separate them.

### reference

https://github.com/element-plus/element-plus/blob/dev/packages/components/avatar/src/avatar.ts#L60-L62